src/components/getFromApi/GetReviewListOfHotel.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from './AxiosConfig'
+
+////id - of object 
+//// loc -location 
+const GetReviewsListOfHotel = (args) =>{
+    return axios.get('https://hotels4.p.rapidapi.com/reviews/list',{
+           params: {
+                id: args
```

Comment:
Do we need to set locale?

Reply:
we discussed in team and as result we resolve to add this feature in future
(means yes we need to set locale)